lib/domain/model/ongoing_call.dart

```diff
@@ -2154,6 +2162,20 @@ class OngoingCall {
             devices.firstWhereOrNull(
               (e) => e.deviceId() == track.getTrack().deviceId(),
             );
+
+        // NoiseSuppression is currently available only on Desktop
+        if (track.isAudioProcessingAvailable() && PlatformUtils.isDesktop) {
+          if (_noiseSuppressionEnabled != null) {
+            await track.getTrack().setNoiseSuppressionEnabled(
+              _noiseSuppressionEnabled!,
+            );
```

Comment:
@ChabanovX, I've noticed that CI fails to build Web platform due to `NoiseSuppressionLevel` enum not being available there. And that seems to be true, if we talk about `NoiseSuppressionLevel` from the `medea_flutter_webrtc`. I've discussed this matter with guys from `medea_flutter_webrtc` and they told me that we shouldn't use this enum from that package - instead the `NoiseSuppressionLevel` from `medea_jason` should be used. Can you please try that? 

And in that case the proper method to change the noise suppression settings would be to call this method over `track` directly:

```suggestion
            await track.setNoiseSuppressionEnabled(
              _noiseSuppressionEnabled!,
            );
```

Reply:
Okay, got it!